### ISSUE TITLE
Add package for formatting package.json

### ISSUE THREAD

**Issue by nedsalk** [FuelLabs/fuels-ts#1289]
We don't have a standardized way to format our `package.json`' contents, which can lead to nits related to it. We should add a package and call it in `pnpm lint`. Perhaps https://www.npmjs.com/package/sort-package-json.

**Comment by arboleya** [comment]
No, this should be a manual thing. Alphabetical sorting has nothing to do with what originated this issue and would de-prioritize everything, probably resulting in a very unpleasant `package.json` file.

**Comment by nedsalk** [comment]
Idk, at first glance [prettier-package-json](https://github.com/cameronhunter/prettier-package-json) does the job pleasantly. They keep pre- and post- prefixes around the command they prefix, which is [very nice](https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.quickmeme.com%2Fimg%2F40%2F40b96f717103244d7d9dea023de22820619058efade0a529f2938430963ed6c7.jpg&f=1&nofb=1&ipt=75f16846e4b5ba1ca742a169d61b42cf67cb38a6a63ecd83fde48344a8efb23e&ipo=images).

**Comment by arboleya** [comment]
Considering it's not alphabetical (bad assumption on my part) _and_ you are fond of it, it may be worth the shot.

It just looks like a low-priority chore when compared to other pressing [issues](https://github.com/FuelLabs/fuels-ts/issues/1298) we have around. No big deal.

**Comment by nedsalk** [comment]
Yes, it's a low prio chore that should be documented as such. I created priority labels for our issues and added one to it.
